packages/cli/test/utils/runUtils.ts

```diff
@@ -17,20 +17,13 @@ export function findApiToken(dirpath: string): string {
 }
 
 export function expectDeepEquals<T>(a: T, b: T, message: string): void {
-  try {
-    expect(a).toEqual(b);
-  } catch (e) {
-    expect.fail(message);
-  }
+  expect(a).toEqualWithMessage(b, message);
```

Comment:
With the simplification we could consider removing this function completely.

Reply:
Could be removed but `toEqualWithMessage` is tagged as deprecated but in the CLI tests we definitely need the message in assertions, having it crossed out everywhere is annoying so I just kept it wrapped in this function. Also less diff to review

Reply:
Oh sure right, somehow I assumed we were iterating object keys here.